txclib/utils.py

```diff
@@ -105,27 +111,57 @@ def determine_charset(response):
 
 def make_request(method, host, url, username, password, fields=None,
                  skip_decode=False):
-    charset = None
-    if host.lower().startswith('https://'):
-        connection = urllib3.connection_from_url(
-            host,
-            cert_reqs=CERT_REQUIRED,
+    # Initialize http and https pool managers
+    num_pools = 1
```

Comment:
As for this one, I believe my code still provides the same functionally with the https:// hostnames (in managers["https"]). Were you suggesting I add additional support?